stores/onboarding/utils/storage.ts

```diff
@@ -0,0 +1,21 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+const ONBOARDING_STORAGE_KEY = '@onboarding_completed';
+
+export const getOnboardingStatus = async (): Promise<boolean> => {
+	try {
+		const status = await AsyncStorage.getItem(ONBOARDING_STORAGE_KEY);
+		return status === 'true';
+	} catch (error) {
+		console.log('AsyncStorage 온보딩 상태 불러오기 실패:', error);
+		return false;
+	}
+};
+
+export const saveOnboardingStatus = async () => {
```

Comment:
보편적인 save메서드와 다르기 때문에 저장될 값을 보여주는 것도 괜찮을 것 같습니다. 😅

```suggestion
export const saveOnboardingAsCompleted = async () => {
```

Reply:
`saveOnboardingAsCompleted`로 변경하는 것이 의미가 더 명확한 것 같습니다. 좋은 의견 감사합니다 🙇

Reply:
반영 감사합니다 🙇‍♂️